llama-index-integrations/llms/llama-index-llms-paieas/tests/test_llms_paieas.py

```diff
@@ -0,0 +1,61 @@
+from llama_index.core.base.llms.base import BaseLLM
+from llama_index.core.base.llms.types import ChatMessage
+from llama_index.llms.paieas import PaiEas
+
+
+def test_text_inference_embedding_class():
+    names_of_base_classes = [b.__name__ for b in PaiEas.__mro__]
+    assert BaseLLM.__name__ in names_of_base_classes
+
+
+def test_pai_eas_llm_complete():
+    eas_llm = PaiEas()
+    if eas_llm:
+        response = eas_llm.complete("What is Alibaba Cloud?")
```

Comment:
we should be skipping these tests if the api key is missing

Reply:
Thanks.